packages/web/src/components/date/DateRange.js

```diff
@@ -212,6 +213,12 @@ class DateRange extends Component {
 		});
 	}
 
+	handleDayMouseEnter = (day) => {
+		this.setState({
+			enteredTo: day,
```

Comment:
This variable isn't being used, did you mean to use `dateHovered`?

Reply:
Yes sorry, I forgot to change this name, it should be `dateHovered` and everything would then be working

Reply:
I tested but it wasn't could you test it out first?